### ISSUE TITLE
Contaminant Screen missing input

### ISSUE THREAD

**Issue by sm1216** [HelikarLab/FastqToGeneCounts#37]
This is the error output

```
Building DAG of jobs...
MissingInputException in line 670 of /lustre/work/helikarlab/sabyasachi/FastqToGeneCounts/Snakefile:
Missing input files for rule contaminant_screen:
results/temp/dump_fastq/eosin_S1R1_complete
```

This is the controls data for eosinophils
```
SRR13729983,eosin_S1R1,PE,total
SRR13730146,eosin_S1R2,PE,total
SRR13730261,eosin_S1R3,PE,total
SRR13730264,eosin_S1R4r1,PE,total
SRR13730265,eosin_S1R4r2,PE,total
SRR13730266,eosin_S1R4r3,PE,total
SRR13730267,eosin_S1R4r4,PE,total
SRR13730276,eosin_S1R5r1,PE,total
SRR13730277,eosin_S1R5r2,PE,total
SRR13730278,eosin_S1R5r3,PE,total
SRR13730279,eosin_S1R5r4,PE,total
SRR8698144,eosin_S2R1r1,SE,mrna
SRR8698145,eosin_S2R1r2,SE,mrna
SRR8698146,eosin_S2R1r3,SE,mrna
SRR8698147,eosin_S2R1r4,SE,mrna
SRR8698148,eosin_S2R1r5,SE,mrna
SRR8698149,eosin_S2R2r1,SE,mrna
SRR8698150,eosin_S2R2r2,SE,mrna
SRR8698151,eosin_S2R2r3,SE,mrna
SRR8698152,eosin_S2R2r4,SE,mrna
SRR8698153,eosin_S2R2r5,SE,mrna
SRR8698154,eosin_S2R3,SE,mrna
SRR8698158,eosin_S2R4,SE,mrna
SRR8698162,eosin_S2R5,SE,mrna
SRR8698166,eosin_S2R6,SE,mrna
SRR8698170,eosin_S2R7,SE,mrna
SRR6829964,eosin_S3R1,PE,total
SRR6829968,eosin_S3R2,PE,total
SRR6829972,eosin_S3R3,PE,total
```

**Comment by JoshLoecker** [comment]
I was able to verify this error using the first line of data on my own system

`SRR13729983,eosin_S1R1,PE,total`

**Comment by JoshLoecker** [comment]
Fix for this coming in #38
